ClearKeep/Scenes/GroupDetail/Views/RemoveMemberView.swift

```diff
@@ -0,0 +1,148 @@
+//
+//  RemoveMember.swift
+//  ClearKeep
+//
+//  Created by MinhDev on 28/03/2022.
+//
+
+import SwiftUI
+import Common
+import CommonUI
+
+private enum Constants {
+	static let spacing = 10.0
+	static let padding = 20.0
+	static let sizeImage = 64.0
+	static let paddingTop = 50.0
+	static let sizeIcon = 18.0
+}
+
+struct RemoveMemberView: View {
+	// MARK: - Constants
+	@Environment(\.colorScheme) var colorScheme
+	@Environment(\.presentationMode) var presentationMode: Binding<PresentationMode>
+
+	// MARK: - Variables
+	@Environment(\.injected) private var injected: DIContainer
+	@Binding var loadable: Loadable<IGroupDetailViewModels>
+	@Binding var member: [GroupDetailProfileViewModel]
+	@State private(set) var groupData: [GroupDetailProfileViewModel] = []
+	@State private(set) var groupId: Int64 = 0
+	@State private(set) var searchStyle: TextInputStyle = .default
+	@State private(set) var searchText: String = ""
+	@State private var messageAlert: String = ""
+	@State private var isShowAlert: Bool = false
+	@State private var isShowPopUp: Bool = false
+	
+	// MARK: - Init
+
+	// MARK: - Body
+	var body: some View {
+		ScrollView(showsIndicators: false) {
+			VStack(alignment: .leading, spacing: Constants.spacing) {
+				SearchTextField(searchText: $searchText,
+								inputStyle: $searchStyle,
+								inputIcon: AppTheme.shared.imageSet.searchIcon,
+								placeHolder: "GroupDetail.Search".localized,
+								onEditingChanged: { isEditing in
+					searchStyle = isEditing ? .highlighted : .normal })
+					.onChange(of: searchText) { text in
+						search(text: text)
+					}
+				ForEach(member) { client in
+					RemoveButtonView(.constant(client.displayName), imageUrl: client.avatar, action: {
+						chosseUser(client) })
+				}
+			}
+		}
+		.padding(.horizontal, Constants.padding)
+		.applyNavigationBarPlainStyle(title: "GroupDetail.RemoveMember".localized,
+									  titleColor: titleColor,
+									  backgroundColors: backgroundButtonBack,
+									  leftBarItems: {
+			BackButtonStandard(customBack)
+		},
+									  rightBarItems: {
+			Spacer()
+		})
+		.background(backgroundColorView)
+		.edgesIgnoringSafeArea(.all)
+		.hiddenNavigationBarStyle()
+		.alert("GroupChat.Warning".localized, isPresented: $isShowPopUp) {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/fe9f474e9dacdb67166a3b18715e917932515a0f/ClearKeep/Scenes/GroupDetail/Views/RemoveMemberView.swift#L71"></span>Arguments should be either on the same line, or one per line.<br />
<code>multiline_arguments</code> <code>RemoveMemberView.swift:71</code></td>
    </tr>
  </tbody>
</table>